src/EditorFeatures/CSharpTest/ConvertToRawString/ConvertToRegularStringToRawStringString_FixAllTests.cs

```diff
@@ -0,0 +1,690 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.CodeRefactorings;
+using Microsoft.CodeAnalysis.CSharp.ConvertToRawString;
+using Microsoft.CodeAnalysis.CSharp.Shared.Extensions;
+using Microsoft.CodeAnalysis.Editor.CSharp.UnitTests.CodeRefactorings;
+using Microsoft.CodeAnalysis.Editor.UnitTests.CodeActions;
+using Microsoft.CodeAnalysis.Test.Utilities;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.Editor.CSharp.UnitTests.ConvertToRawString
+{
+    public class ConvertToRegularStringToRawStringString_FixAllTests : AbstractCSharpCodeActionTest
+    {
+        protected override CodeRefactoringProvider CreateCodeRefactoringProvider(Workspace workspace, TestParameters parameters)
+            => new ConvertRegularStringToRawStringCodeRefactoringProvider();
+
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsFixAllOccurrences)]
+        public async Task FixAllInDocument_SingleLine()
```

Comment:
First three tests verify FixAllInDocument + the fact that equivalence key is taken into consideration while computing the FixAll. Rest of the tests verify different FixAll scopes, but only for `ConvertToRawKind.SingleLine`